frontend/src/scenes/teamLogic.tsx

```diff
@@ -33,11 +46,12 @@ export const teamLogic = kea<teamLogicType<TeamType>>({
                         return null
                     }
                 },
-                patchCurrentTeam: async (patch: Partial<TeamType>) => {
+                updateCurrentTeam: async (payload: Partial<TeamType>) => {
                     if (!values.currentTeam) {
                         throw new Error('Current team has not been loaded yet, so it cannot be updated!')
                     }
-                    const patchedTeam = (await api.update(`api/projects/${values.currentTeam.id}`, patch)) as TeamType
+                    actions.setUpdatingTeamPayload(payload)
+                    const patchedTeam = (await api.update(`api/projects/${values.currentTeam.id}`, payload)) as TeamType
```

Comment:
This can lead to funny edge cases --> e.g. if you're clicking some things on the settings page faster than your 2.5G connection allows, the first one might still be pending when you start the second one (and overwrite what's in the `updateCurrentTeam` reducer).

In this case I'd suggest just moving all the toasting code from `updateCurrentTeam` into here. It should remove the need for this reducer altogether...

Reply:
Agreed, addressed!